cinema/views.py

```diff
@@ -183,7 +234,7 @@ class OrderViewSet(
     )
     serializer_class = OrderSerializer
     pagination_class = OrderPagination
-    authentication_classes = (TokenAuthentication,)
+
```

Comment:
The `permission_classes` attribute is correctly set to `IsAuthenticated`, which aligns with the intended access control for this viewset.